cluster-provision/k8s/1.17/manifests/cnao/operator.yaml

```diff
@@ -121,7 +121,7 @@ apiVersion: apps/v1
 kind: Deployment
```

Comment:
Please limit the change only to 1.19+, where we run CNAO. We can't assure that this new version of CNAO is compatible with old versions of Kubernetes.